### ISSUE TITLE
Error on cursor move after scala validation

### ISSUE THREAD

**Issue by yazgoo** [ensime/ensime-vim#65]
````
Error detected while processing function ensime#au_cursor_moved..<SNR>35_call_plugin:

line   12:
E15: Invalid expression: ;' expected but identifier found.'
Press ENTER or type command to continue
Error detected while processing function ensime#au_cursor_moved..<SNR>35_call_plugin:

line   12:
E15: Invalid expression: ;' expected but identifier found.'
Press ENTER or type command to continue
Error detected while processing function ensime#au_cursor_moved..<SNR>35_call_plugin:

line   14:
E605: Exception not caught: vim.error:None
Press ENTER or type command to continue
````

**Comment by yazgoo** [comment]
The problem comes from not escaping quotes in message() method 

```python
self.vim.command("echo '{}'".format(m))
```

**Comment by yazgoo** [comment]
fix in 642130ff06beb4b8fad9099b1a4469787358d16e
